### PR TITLE
add toco ontology

### PR DESCRIPTION
**Pull request by QianruZhou333**

### CODE REVIEW

**Comment by dgarijo** [comment]
@QianruZhou333,
your readme file does not seem correct. Can you complete it with a brief description and stating the maintainers pls?

**Comment by davidlehn** [comment]
Looks like the README.md file is a zip archive.  There's a preview jpg in there that looks like what the README.md should contain.  When you update this, please get the old large zip file out of the git history.  (@dgarijo If that doesn't happen, we should squash it away)

File: toco/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# /TOUCAN Ontology (ToCo) / 
+This [W3ID](https://w3id.org) provides a persistent URI namespace for TOUCAN Ontology (ToCo) resources.
+
+ToCo is an ontology for hybrid telecommunication networks. An Device-Interface-Link (DIL) ontology design pattern is proposed for the first time to our knowledge.
```
Comment (TallTed) [inline toco/README.md:4]:
```suggestion
ToCo is an ontology for hybrid telecommunication networks. A Device-Interface-Link (DIL) ontology design pattern is proposed for the first time to our knowledge. 
```

File: toco/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+This [W3ID](https://w3id.org) provides a persistent URI namespace for TOUCAN Ontology (ToCo) resources.
+
+ToCo is an ontology for hybrid telecommunication networks. An Device-Interface-Link (DIL) ontology design pattern is proposed for the first time to our knowledge. 
+
+Have been published on LOV (Linked Open Vocabulary) with the entry: https://lov.linkeddata.es/dataset/lov/vocabs/toco
+
+This development is part of an on-going project which is addressing the convergence of telecommunication networks across multiple technology domains. 
+The DIL pattern is observed and summarized through the ontology developing progress. It could describe all kinds of networks. 
+
+Build around the DIL pattern, the ToCo ontology describes the physical infrastructure, quality of channel, services and users in heterogeneous telecommunication networks which span multiple technology domains across the networks. Examples and use cases of ToCo in recent projects are discussed in publications.
```
Comment (TallTed) [inline toco/README.md:11]:
```suggestion
Built around the DIL pattern, the ToCo ontology describes the physical infrastructure, quality of channel, services, and users in heterogeneous telecommunication networks which span multiple technology domains across the networks. Examples and use cases of ToCo in recent projects are discussed in publications.
```

File: toco/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## How to contribute 
+####  $ git init
+####  $ git clone https://github.com/QianruZhou333/toco_ontology.git
```
Comment (TallTed) [inline toco/README.md:15]:
```suggestion
####  `$ git init`
####  `$ git clone https://github.com/QianruZhou333/toco_ontology.git`
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor grammar and punctuation tweaks

**Comment by dgarijo** [comment]
Inactive PR. Closing, please reopen if appropriate

**Pull request closed without merging**